alca-app/src/pages/Positions/Positions.jsx

```diff
@@ -95,6 +95,88 @@ export function Positions() {
         },
     ];
 
+    const lockedPositionsColumns = [
+        {
+            field: "amount",
+            headerName: "Amount",
+            flex: 0.5,
+            sortable: false,
+            headerClassName: "headerClass",
+        },
+        {
+            field: "id",
+            headerName: "ID",
+            flex: 0.25,
+            sortable: false,
+            headerClassName: "headerClass",
+        },
+        {
+            field: "lockedDate",
+            headerName: "Locked Date",
+            flex: 0.5,
+            sortable: false,
+            headerClassName: "headerClass",
+        },
+        {
+            field: "timeLeft",
+            headerName: "Time Left",
+            flex: 0.5,
+            sortable: false,
+            headerClassName: "headerClass",
+        },
+        {
+            field: "rewardsAchieved",
+            headerName: "Rewards Achieved",
+            flex: 0.75,
+            sortable: false,
+            headerClassName: "headerClass",
+            renderCell: (params) => (
+                <Box sx={{ width: "100%" }}>
+                    <LinearProgress variant="determinate" color="secondary" value={40} />
+
+                    <Box sx={{ fontSize: "10px", fontFamily: theme.typography.fontFamily, marginTop: 0.7 }}>
```

Comment:
Should use numbers instead of strings (frontSize: "10px" => fontSize: 10) for fixed size attributes.